packages/core/src/sequelize.js

```diff
@@ -304,6 +304,10 @@ export class Sequelize extends SequelizeTypeScript {
       this.options.logging = console.debug;
     }
 
+    if (this.options.quoteIdentifiers === false && (this.options.dialect === 'postgres' || this.options.dialect === 'snowflake')) {
```

Comment:
shouldn't this warn on all dialects?

Reply:
It should indeed, although it doesn't do anything on other dialects. Fixed